docs/examples/barcode-image/index.html

```diff
@@ -47,23 +80,29 @@ <h1 class="title">Scan barcode from Image</h1>
 		</main>
 
 		<script type="text/javascript" src="../../../bundles/library.min.js"></script>
-        <script type="text/javascript">
+		<script type="text/javascript">
 			window.addEventListener('load', function() {
 				const codeReader = new Library.BrowserBarcodeReader('video');
 				console.log('ZXing code reader initialized');
 
-				document.getElementById('decodeButton').addEventListener('click', () => {
-					const img = document.getElementById('img');
+				const decodeButtons = document.getElementsByClassName("decodeButton");
+
+				const decodeFun = function(ev) {
+					const parent = this.parentNode.parentNode;
+					const img = parent.getElementsByClassName('img')[0].cloneNode(true);
```

Comment:
Here I used clonenode because in the previous version after the first click the image disappeared